cleanlab/object_detection/summary.py

```diff
@@ -34,6 +42,184 @@
     Image = TypeVar("Image")
 
 
+def object_counts_per_image(
+    labels=None,
+    predictions=None,
+    *,
+    auxiliary_inputs=None,
+) -> Tuple[List, List]:
+    """Return the number of annotated and predicted objects for each image in the dataset.
+
+    This method can help you discover images with abnormally many/few object annotations.
+
+    Parameters
+    ----------
+    labels :
```

Comment:
Can you hyperlink the format of labels/predictions for all functions but one in this page similarly to what is done [here](https://github.com/cleanlab/cleanlab/blob/6d45971cf83ccc85ac31d2591c54ce6b52a27281/cleanlab/object_detection/rank.py#L648)? (This way if we change anything in the format the update will only have to happen once per file)

I suggest linking labels/predictions definitions of `bounding_box_size_distribution` and `class_label_distribution` to the descriptions written out in `object_counts_per_image` since that appears first.